### PR TITLE
Fix V2 positions

### PR DESCRIPTION
**Pull request by KMKoushik**

### CODE REVIEW

File: packages/frontend/src/hooks/contracts/useTokenBalance.ts [hunk 1]
```diff
@@ -71,15 +73,11 @@ async function updateBalance(
   address: string | null,
   decimals: number,
 ) {
-  try {
```
Comment (daryakaviani) [inline packages/frontend/src/hooks/contracts/useTokenBalance.ts:74]: Why are we removing the try-catch here?

**Review by daryakaviani — APPROVED** [review]
lgtm - i am able to switch from mainnet to ropsten and see all my ropsten positions

**Comment by daryakaviani** [comment]
Perhaps a separate issue, but when switching from Ropsten -> Mainnet, the Crab V2 position shows as loading even though I do not have a Crab V2 position on Mainnet:
image

**Review by alexisgauba — APPROVED** [review]

**Comment by KMKoushik** [comment]
> Perhaps a separate issue, but when switching from Ropsten -> Mainnet, the Crab V2 position shows as loading even though I do not have a Crab V2 position on Mainnet: image

This should be fixed once the mainnet is migrated, if not we can come back to this

**Pull request merged**